tutorials/arvo/parsing.md

```diff
@@ -475,12 +481,17 @@ as the bunt value of the type that the `rule` produces, as well as a gate that
 accepts a `rule`. `+knee` produces a `rule` that implements any recursive calls
 in the `rule` in a manner acceptable to the compiler. Thus the preferred manner
 to write the above `rule` is as follows:
+
 ```hoon
-|-(;~(plug prn ;~(pose (knee *tape |.(^$)) (easy ~))))
+++  pars
+  |-
+  ;~  plug  prn
+    ;~  pose  knee  *tape
+      |.  ^$  (easy ~)
```

Comment:
```suggestion
++  pars
  |-
  ;~  plug  prn
    ;~  pose
      (knee *tape |.(^$))
      (easy ~)
    ==
  ==
```

btw, I think it's more common to wrap `+knee` around the parser on the outside (ie, directly under the `|-` here, and as we also do below). This has the helpful effect of specifying a parsing result type early on. I don't think it technically matters much though.